src/JsonAsynchronousNodeKit/PropertyHandler{T}.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace JsonAsynchronousNodeKit;
+
+public record PropertyHandler<T>(ParameterType ParameterType)
```

Comment:
What do you think about moving this into `PropertyHandler.cs`? I'm not a fan of having separate file for just one line :)

Reply:
I'll have to suppress SA1402 but I think it makes sense in this instance. The Type-less version only exists so we can have a base-type to give the dictionary.